docs/README.md

```diff
@@ -110,10 +110,10 @@ than statically compiled into kpt.*
 
 Reconcile configuration files with the live state.
 
-- Apply a package
+- Apply package resources to live cluster
```

Comment:
I don't think we should use the phrase 'live cluster'. We do refer to the state of the cluster as the 'live state'.

Reply:
Done